execution_engine/src/core/engine_state/step.rs

```diff
@@ -39,13 +39,25 @@ impl RewardItem {
     }
 }
 
+#[derive(Debug)]
+pub struct DisableItem {
```

Comment:
I've only thought of calling it `EvictItem` after I pushed the changes :stuck_out_tongue: I think I like it better, but since it's on the EE side, anyway, I'll let the EE guys choose the name they prefer :wink:

Reply:
EvictItem